test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.cs

```diff
@@ -98,6 +98,7 @@ public class HubConnectionTests
 
             await hubConnection.StartAsync();
             await hubConnection.DisposeAsync();
+            await hubConnection.Closed.OrTimeout();
```

Comment:
this seems wrong - hubConnection has been disposed so the expectation that the `Closed` task is valid is not correct. The bug is somewhere else...

Reply:
Yeah I wasn't sure about that. We could await the closed task in DisposeAsync?

Reply:
Yes, it has to be something to this effect